src_kolko_krzyżyk/TicTacToe_expand.py

```diff
@@ -185,7 +185,35 @@ def main(self):
             else:
                 player = 0
 
-game = TicTacToe(5)
+def menu():
+    print('''\n\n\t"Simple game for two person in Tic Tac Toe.
+     Player decides how big the board is. Valid board (2 to n)
+     where board are matrix n x n. Good Luck and Have Fun."\n\n''')
+    # default matrix_choice
+    matrix_choice = 3
+    while True:
+        matrix_choice = input('Please enter size of martix from two to n: ')
+        try:
+            matrix_choice = int(matrix_choice)
+            break
+        except:
+            print("Dosen't entered number please try again" )
+
+    while True:
```

Comment:
As like above. You can move asking user of wining condition to a different function and place here code like:
```
wining_condition = set_wining_condition()
return TicTacToe(matrix_size, wining_condition).main()
```